File: itead_example/py_test/digital.py
import wiringIO as IO
import sys

pin =134

IO.pinMode(pin,IO.OUTPUT)

Mo = IO.pinMode(pin,IO.CHECK)
MS = 1000

print "**********************"
print "delay:%s ms,Mode:%s" %(MS,Mo)
print "**********************"

while 1:
	IO.digitalWrite(pin,1)
	val = IO.digitalRead(pin)
        print "LED on val:%d" %(val)
	IO.delay(MS)
	IO.digitalWrite(pin,0)
	val = IO.digitalRead(pin)
        print "LED off val:%d" %(val)
	IO.delay(MS)

File: itead_example/c_test/digital.c
/**
 * softPwm test 
 * author:gootoomoon 
 * gcc -o test digital.c -lwiringPi -lpthread
 */
#include <stdio.h>
#include <errno.h>
#include <string.h>

#include <itead.h>

int pin = 134;

int main()
{
	pinMode(pin,OUTPUT);

	for(;;){
		digitalWrite(pin,HIGH);
		delay(1000);
		digitalWrite(pin,LOW);
		delay(1000);
	}
}
